docs/collecting-data/code-generation/client-side-validation/index.md

```diff
@@ -57,13 +57,59 @@ As we can observe, the value passed as the `id` attribute is violating the schem
 
 Currently the validation information will include attributes that can help point out the issue happening at the schema level, plus the stack trace revealing the caller of the function.
 
+## Entity cardinality rules validation example
+
+:::info
+This feature is available since version 0.3.1 of Snowtype for the [Browser Tracker](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/quick-start-guide/?platform=browser) in TypeScript.
```

Comment:
nit: maybe we do not need to specify `in TypeScript`.

Reply:
It is not available in JavaScript templates yet, that is why the clarification.